HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -4246,4 +4246,46 @@ def self.get_dehumidifier_default_values(capacity)
 
     return { rh_setpoint: rh_setpoint, ief: ief }
   end
+
+  def self.get_default_hvac_efficiency(year, hvac_type, fuel_type, units)
+    year = 0 if year.nil?
```

Comment:
I agree that it will always have a non-nil value unless a user provides efficiency. I guess we can remove it.

Reply:
Just to be clear, the point here is that even if year was nil, assuming zero would be a _very_ poor choice as it would end up choosing the default equipment efficiency based on the first year available in the lookup tables (so, 1970).

Reply:
Ah, good point. I tried to keep the method the way it is. But I should have thought about it critically.

Reply:
I see. That made me concerned about the original code, but I looked into the HEScore code and it appears year is only nil when it's an energystar efficiency lookup (so basically the year doesn't matter and is set to zero just to avoid an error). So that's why that is appropriate in the original code but not here.